### PR TITLE
Remove assert statements from main code and replace with raise error

### PR DESCRIPTION
**Pull request by elehmer**
This simple PR replaces `assert` statements used in main code and replaces them with `raise` statements. The test logic is reversed to do this.

I left the `assert` statements in the test files as it seem appropriate there.

### CODE REVIEW

File: climakitae/selectors.py [hunk 1]
```diff
@@ -721,10 +721,8 @@ def _display_select(selections, location, location_type = "area average"):
     for location_type -- 'area average' is the only choice, until station-based
     data are available.
     """
-    assert location_type in [
-        "area average",
-        "station",
-    ], "Please enter either 'area average' or 'station'."
+    if location_type not in ["area average", "station"]:
+        raise ValueError("Please enter either 'area average' or 'station'.")
```
Comment (nicolejkeeney) [inline climakitae/selectors.py:725]: Glad you noted this as I've been meaning to come back to this. We don't actually ever use this code; I think its old placeholder code from many months ago. Should we just delete these lines?

Comment (elehmer) [reply climakitae/selectors.py:725]: Yes! Reducing unneeded code would be great!

Comment (nicolejkeeney) [reply climakitae/selectors.py:725]: Will do!

**Review by nicolejkeeney — APPROVED** [review]
I removed the old code. Looks good to me and tested and it runs fine. Can you double check that it works before merging?

**Comment by elehmer** [comment]
Is this ready to merge?

**Comment by elehmer** [comment]
> I removed the old code. Looks good to me and tested and it runs fine. Can you double check that it works before merging?

Works...merging...

**Pull request merged**